command/genesis/params.go

```diff
@@ -481,41 +481,55 @@ func (p *genesisParams) extractNativeTokenMetadata() error {
 			Symbol:     defaultNativeTokenSymbol,
 			Decimals:   defaultNativeTokenDecimals,
 			IsMintable: false,
+			Owner:      types.ZeroAddress,
 		}
 
 		return nil
 	}
 
 	params := strings.Split(p.nativeTokenConfigRaw, ":")
-	if len(params) != nativeTokenParamsNumber {
+	if len(params) < minNativeTokenParamsNumber {
 		return errInvalidTokenParams
 	}
 
+	// name
 	name := strings.TrimSpace(params[0])
 	if name == "" {
 		return errInvalidTokenParams
 	}
 
+	// symbol
 	symbol := strings.TrimSpace(params[1])
 	if symbol == "" {
 		return errInvalidTokenParams
 	}
 
+	// decimals
 	decimals, err := strconv.ParseUint(strings.TrimSpace(params[2]), 10, 8)
 	if err != nil || decimals > math.MaxUint8 {
 		return errInvalidTokenParams
 	}
 
+	// is mintable native token used
 	isMintable, err := strconv.ParseBool(strings.TrimSpace(params[3]))
 	if err != nil {
 		return errInvalidTokenParams
 	}
 
+	// in case it is mintable native token, it is expected to have 5 parameters provided
+	if isMintable && len(params) != minNativeTokenParamsNumber+1 {
+		return errInvalidTokenParams
+	}
+
+	// owner address
```

Comment:
i think this will fail if `!isMintable` and fifth parameter is not provided

Reply:
Spot on 🙌 : https://github.com/0xPolygon/polygon-edge/pull/1603/commits/b0a999f940e0f5fdcca105728aa7ccec0b45f2f7.

Implemented unit test as well: https://github.com/0xPolygon/polygon-edge/pull/1603/commits/1c008c19ed981455201429e5a61a48a411fc8851